samza-core/src/main/java/org/apache/samza/runtime/LocalApplicationRunner.java

```diff
@@ -179,20 +189,51 @@ public ApplicationStatus status(StreamApplication streamApp) {
   }
 
   /**
+   * Generates processorId for each {@link StreamProcessor} using the configured {@link ProcessorIdGenerator}
+   *
+   * @param config Application config
+   * @return String that uniquely represents an instance of {@link StreamProcessor} in the current JVM
+   */
+  /* package private */
+  String getProcessorId(Config config) {
```

Comment:
Same question as in PR 117. Shouldn't JobCoordinator be generating the processorId according to SEP-1? Is this temporary? Same with passing processorId as a constructor param to StreamProcessor.